librz/core/cmd_eval.c

```diff
@@ -502,14 +502,14 @@ RZ_IPI int rz_cmd_eval(void *data, const char *input) {
 	RzCore *core = (RzCore *)data;
 	switch (input[0]) {
 	case '\0': // "e"
-		rz_config_list(core->config, NULL, 0);
```

Comment:
@officialcjunior @ret2libc I think we can safely remove this `rz_cmd_eval()` function since the `e` commands were completely ported to the newshell, and there is no `RZ_CMD_DESC_OLDINPUT` for `e` commands in `librz/core/cmd_descs/cmd_descs.yaml`.

Reply:
Yes, I think it's about time that function can be removed. 👍🏽 
Will wait for what ret2libc has to say.

Reply:
Removed.